grazer/src/main/java/edu/asu/diging/grazer/core/domain/ITransformationFiles.java

```diff
@@ -0,0 +1,23 @@
+package edu.asu.diging.grazer.core.domain;
+
+import org.springframework.web.multipart.commons.CommonsMultipartFile;
+
+/**
+ * 
+ * This class stores the file names and file data of one pattern file and its 
+ * corresponding transformation file. 
+ * 
+ * @author mshah18
+ *
+ */
+public interface ITransformationFiles {
+    
+    CommonsMultipartFile getTransformationFile();
+    
+    void setTransformationFile(CommonsMultipartFile transformationFile);
+    
+    CommonsMultipartFile getPatternFile();
```

Comment:
assuming that this interface belongs to the service/logic layer of the app, there should be no dependencies to the web layer (which using CommonsMultipartFile is).